ppml/src/main/cpp/quote_verification/App/quote_verification.cpp

```diff
@@ -0,0 +1,244 @@
+#include <stdio.h>
```

Comment:
App is not necessary. 
We can use this path https://github.com/intel-analytics/BigDL-core/tree/master/ppml/src/main/java/com/intel/analytics/bigdl/ppml/attestation